imagersite/imager_images/templates/imager_images/library.html

```diff
@@ -0,0 +1,28 @@
+{% extends "imagersite/base.html" %}
+{% load static %}
+
+
+{% block nav-bar %}
+<a href="{% url 'home' %}" class="btn btn-primary btn-lg">Home</a>
+    {% if user.is_authenticated %}
+        <a href="{% url 'logout' %}" class="btn btn-primary btn-lg">Logout</a>
+        <a href="{% url 'user_profile' %}" class="btn btn-primary btn-lg">{{ user.username }}</a>
+    {% else %}
+        <a href="{% url 'registration_register' %}" class="btn btn-primary btn-lg">Register</a>
+        <a href="{% url 'login' %}" class="btn btn-primary btn-lg">Login</a>
+    {% endif %}
+{% endblock %}
+
+
+{% block content %}
+
+<div class="">
+<h3>Photos</h3>
```

Comment:
The library page should include a grid of all photos with the title of owned photos of the logged in user and all albums.